packages/admin/cms-admin/src/dam/CurrentDamFolderContext.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+
+type FolderContextType = {
```

Comment:
Please rename to CurrentDamFolderContext

Reply:
Also, why is this an object, and not a simple string?

Reply:
I felt like this would be beneficial for code readability.